pages/getData.tsx

```diff
@@ -103,6 +103,92 @@ const GetData: NextPage = () => {
     setData(data);
   };
 
+  const decodeData = (data: string) => {
```

Comment:
Maybe "decodePermissionsData"? 

Can't you use this instead:

https://docs.lukso.tech/tools/erc725js/classes/ERC725#decodepermissions

?